pkg/chains/formats/slsa/internal/resolved_dependencies/resolved_dependencies_test.go

```diff
@@ -17,13 +17,34 @@ limitations under the License.
 package resolveddependencies
 
 import (
+	"encoding/json"
+	"strings"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/common"
 	v1slsa "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v1"
+	"github.com/tektoncd/chains/internal/backport"
+	"github.com/tektoncd/chains/pkg/artifacts"
+	"github.com/tektoncd/chains/pkg/chains/formats/slsa/internal/slsaconfig"
+	"github.com/tektoncd/chains/pkg/chains/objects"
+	"github.com/tektoncd/chains/pkg/internal/objectloader"
+	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
+	logtesting "knative.dev/pkg/logging/testing"
 )
 
+const digest = "sha256:05f95b26ed10668b7183c1e2da98610e91372fa9f510046d4ce5812addad86b7"
+
+var pro *objects.PipelineRunObjectV1
+var proStructuredResults *objects.PipelineRunObjectV1
+
+func init() {
+	pro = createPro("../../testdata/slsa-v2alpha3/pipelinerun1.json")
```

Comment:
Consider creating `pro` and `proStructuredResults` as local variables where they are needed to avoid tests accidentally influencing others.

Reply:
Done!